Marlin/Marlin.h

```diff
@@ -231,8 +231,10 @@ enum DebugFlags {
 extern uint8_t marlin_debug_flags;
 
 extern bool Running;
+extern volatile bool SafetyCriticalSection;
```

Comment:
When I was originally testing, I was afraid that this was getting cached but I think I found out what was happening there. This should be safe to remove.